manager/api/transport.go

```diff
@@ -14,6 +15,8 @@ import (
 	"github.com/prometheus/client_golang/prometheus/promhttp"
 )
 
+const MAX_LIMIT_SIZE = 100
```

Comment:
See the point given above. Also, **never use java conventions for naming go constants**.